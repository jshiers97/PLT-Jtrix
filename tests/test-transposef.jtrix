void main() {
	Matrix<float> a;
	float[] dim;
	int i;
	int j;
	
	a = [[1.];[2.]];

	a = a.transpose_f();

	dim = a.dim_f();

	for(i = 0; i < f_to_int(dim[0]); i = i++) {
		for(j = 0; j < f_to_int(dim[1]); j = j++) {
			printf(a[i][j]);
		}
	}

	println("r x c");
	print(f_to_int(dim[0]));
	print(f_to_int(dim[1]));

	free(dim);
	free(a);

}
