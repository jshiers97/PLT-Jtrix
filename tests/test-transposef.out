1.000
2.000
r x c
1
2
